src/_resources/fact-sheet-enhancing-transit-experience-with-bankon.md

```diff
@@ -0,0 +1,32 @@
+---
+date: "2023-10-01T17:00:00-07:00"
+title: "Fact Sheet: Enhancing the transit experience with BankOn accounts"
```

Comment:
Let's use sentence case here like the others `Fact sheet: Enhancing...`

Reply:
Ah, thanks for catching